elks/arch/i86/lib/memmove.c

```diff
@@ -1,29 +0,0 @@
-/*******************************************************************
- * memmove function for far data. The number of bytes transferred  *
- * must be <= 32767 ( ie. fit in an signed int ). That's because   *
- * of ( you guessed it... ) the braindead segmented architecture   *
- * Saku Airila 1996                                                *
- *******************************************************************/
-
-#include <linuxmt/mm.h>
-
-void blt_forth(unsigned,unsigned,unsigned,unsigned,unsigned);
-
-void far_memmove(unsigned sseg, unsigned soff, unsigned dseg, unsigned doff,
```

Comment:
I assume that `fmemcpy` already does the same thing as this.

Reply:
Not exactly, `fmemmove` manages the overlap, not `fmemcpy`. According to the documentation, the first was used in the console for scrolling, but is not used any more, so I removed it from the tree.